titan-web-client/src/layouts/dashboard/components/sidebar/SidebarHeading.js

```diff
@@ -17,7 +17,7 @@ class SidebarHeading extends React.Component {
 }
 
 SidebarHeading.propTypes = {
-  children: PropTypes.object
+  children: PropTypes.string

```

Comment:
`children`  shouldn't be restricted to a string value. We may want to render more complex components as the heading text (i.e notification indicator, expand/collapse toggle, etc.).

Reply:
As the following review mentions, `children` shouldn't need to be provided here anymore.